File: Data Structures & Algorithms/maximum-product-subarray/submission-2.java
class Solution {
    public int maxProduct(int[] nums) {
        int n=nums.length;
        if(n==1) return nums[0];
        int res=0;
        for(int i=0;i<n;i++){
            int curr=1;
            int j=i;
            while(j<n){
                curr*=nums[j];
                res=Math.max(res,curr);
                j++;
            }
        }
        return res;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums=sorted(nums)
        longets=0
        L=[0]
        for i in range(len(nums)-1) :
            if nums[i+1]-nums[i] == 1 :
                longets+=1
            elif nums[i+1] == nums[i]:
                longets+=0
            else :
                L.append(longets+1)
                longets=0
        L.append(longets+1)
        if len(nums) == 0 :
            return 0
        else : 
            return max(L)


